common/src/main/res/values/strings.xml

```diff
@@ -733,10 +737,14 @@
     <string name="shortcut5_note">Not all launchers support displaying 5 shortcuts, you may not see this shortcut if the above 4 are already added.</string>
     <string name="shortcuts_summary">Add shortcuts to launcher</string>
     <string name="shortcuts_tile">Shortcuts tile</string>
+    <string name="shortcut_tiles">Shortcut tiles</string>
+    <string name="shortcuts_tile_n">Shortcuts tile #%d</string>
+    <string name="shortcuts_tile_select">Select shortcuts tile to manage</string>
     <string name="shortcuts_tile_description">Select up to 7 entities</string>
     <string name="shortcuts_tile_empty">Choose entities in settings</string>
     <string name="shortcuts_tile_text_setting">Show names on shortcuts tile</string>
     <string name="shortcuts_tile_log_in">Log in to Home Assistant to add your first shortcut</string>
+    <string name="shortcuts_tile_no_tiles_yet">There are not Shortcuts tiles added yet.</string>
```

Comment:
> not Shortcuts tiles added yet

- I think that `t` isn't needed?
- "Shortcuts tiles" is inconsistent with the option "Shortcut tiles" that was previously used
- Drop the period `.` here ([style guidance](https://m3.material.io/foundations/content-design/style-guide/grammar-and-punctuation#555b17c7-de48-4974-8a82-51106899dc0e), or compare with other empty states in the app)

Reply:
Fixed in 91a6de7a